impl/src/planning.rs

```diff
@@ -133,6 +135,7 @@ struct CrateSubplanner<'planner> {
 pub struct PlannedBuild {
   pub workspace_context: WorkspaceContext,
   pub crate_contexts: Vec<CrateContext>,
+  pub buildfile_suffix: String,
```

Comment:
Nit: I think RenderDetails is a better place for this field. It is currently initialized [here](https://github.com/google/cargo-raze/blob/30fb8b43192858316272b3a0e79c9b2ef86ed0bb/impl/src/bin/cargo-raze.rs#L134-L136), and the suffix can be extracted from the settings at that location.